ceiling-tests/v2/README.md

```diff
@@ -0,0 +1,606 @@
+  "AmazonS3Bucket": "agm-ct-bucket-20230829-01",
+  "AmazonS3PathPrefix": "",
+  "AmazonS3Region": "us-east-1",
+  "AmazonS3Endpoint": "s3.amazonaws.com",
+  "AmazonS3SSL": true,
+  "AmazonS3SignV2": false,
+  "AmazonS3SSE": false
+},
+```
+
+All tests were designed to stress the latest [ESR version](https://docs.mattermost.com/upgrade/extended-support-release.html) at the time of testing: v9.5.7.
+
+The license used was an internal one used by developers for tests, but any Enterprise license would do.
+
+``` json
+"MattermostDownloadURL": "https://releases.mattermost.com/8.1.7/mattermost-8.1.7-linux-amd64.tar.gz",
+"MattermostLicenseFile": "/home/ubuntu/licenses/lt_enterprise-500K.mattermost-license",
+```
+
+The version of the load-test tool used by agents was v1.18.0. See the [Load-test tool version](#load-test-tool-version) section to know more about it.
+
+``` json
+"LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.18.0/mattermost-load-test-ng-v1.18.0-linux-amd64.tar.gz",
+```
+
+The final settings do not affect the results of the tests, but are needed to allow for disk usage for each type of instance. The `Metrics` and `ElasticSearch` disks were modified for different tests, ranging from 50 to 200GiB. Pyroscope's profiling for agents was disabled for the larger tests, since it created large amounts of data:
+
+``` json
+"StorageSizes": {
+  "Agent": 10,
+  "Proxy": 10,
+  "App": 10,
+  "Metrics": 200,
+  "ElasticSearch": 200
+},
+"PyroscopeSettings": {
+  "EnableAppProfiling": true,
+  "EnableAgentProfiling": false
+}
+```
+
+### Mattermost
+#### Mattermost version
+
+The tested version of Mattermost was [v9.5.7](https://github.com/mattermost/mattermost/releases/tag/v9.5.7).
+
+#### Architecture
+
+- Application:
+  - The first test use a single-node deployment.
+  - All other tests use High Availability deployments, with two or more application nodes deployed in a cluster with a proxy node on front acting as the entry point and sharing the requests among all app nodes. 
+- Elasticsearch: Tests with Elasticsearch enabled always used 2 instances of the same instance type.
+- Job servers; the deployment architecture omitted [job servers](https://docs.mattermost.com/scale/high-availability-cluster.html#job-server). Tests did not stress data-intensive tasks like LDAP syncing or data retention either.
+- Database: All deployments have the database running in a different server, considering both:
+  - Single-node DB deployments.
+  - Multiple node DB deployments, with a writer instance and the rest acting as reader replicas.
+
+#### Proxy
+
+All deployments with more than one app node had a proxy acting as a load balancer:
+- Specs: the proxy ran in an `m7i.4xlarge` instance for the lower-end tests and in an `m7i.8xlarge` instance for the higher-end ones.
+- Version: the proxy ran `nginx v1.27.1`.
+- Configuration: the proxy was configured with the following settings:
```

Comment:
Should we add a note that we are not testing TLS offloading?

Reply:
Added a note in the Architecture section a bit above. See https://github.com/mattermost/performance-reports/pull/22/commits/f43d0e41ffb270757875997d5d396b28701f6a75